cmd/api/src/database/agi.go

```diff
@@ -43,9 +46,18 @@ func (s *BloodhoundDB) CreateAssetGroup(ctx context.Context, name, tag string, s
 		err error
 	)
 
-	err = s.AuditableTransaction(ctx, auditEntry, func(tx *gorm.DB) error {
-		return CheckError(tx.Create(&assetGroup))
-	})
+    err = s.AuditableTransaction(ctx, auditEntry, func(tx *gorm.DB) error {
+        err := tx.Create(&assetGroup).Error
+        if err != nil {
+            log.Infof("Error creating asset group: %v", err)
```

Comment:
Is this log intentional or just part of debugging?

Reply:
Debugging - removed.